app/src/main/java/com/smap/f16/grp12/racketometer/fragments/HistoryFragment.java

```diff
@@ -0,0 +1,118 @@
+package com.smap.f16.grp12.racketometer.fragments;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.ListView;
+
+import com.smap.f16.grp12.racketometer.R;
+import com.smap.f16.grp12.racketometer.adaptors.SessionAdaptor;
+import com.smap.f16.grp12.racketometer.models.Session;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * A fragment representing a list of Items.
+ * <p>
+ * Activities containing this fragment MUST implement the {@link OnListFragmentInteractionListener}
+ * interface.
+ */
+public class HistoryFragment extends Fragment {
+
+    private ListView lstView;
+    private ArrayList<Session> sessions;
+    private SessionAdaptor sessionAdaptor;
+    private OnListFragmentInteractionListener mListener;
+
+    private static final String SESSIONS = "SESSIONS";
```

Comment:
This is not likely to stay unique.
Prefered usage is to write the full package indentifier like:
com.smap.f16.grp12.racketometer.fragments.HistoryFragment.SESSIONS